### ISSUE TITLE
Catmandu command line sort doesn't work for MongoDB Store

### ISSUE THREAD

**Issue by phochste** [LibreCat/Catmandu-Store-MongoDB#12]
The `sort` in the code needs to be a blessed reference. But I would like to be able to specify:

```
$ catmandu export mydatabase --query '{"foo":"bar"}' --sort '{"timestamp":1}'
```

**Comment by jorol** [comment]
fixed in pr #13

**Comment by jorol** [comment]
released in with version 0.0803
